portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,44 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav>
+      <label class="logo">KvngSnow's Portfolio</label>
+      <ul>
+          <li><a href="http://www.github.com/JalenJenkins">Github</a></li>
+          <li><a href="https://www.instagram.com/kvngsno">Instagram</a></li>
+          <li><a href="https://www.linkedin.com/in/jalen-jenkins-b75772186">Linkedin</a><li> 
+          </ul>
+
+    <div id="image"> <img src = "images/self image.jpg" alt = "A picture of myself" style="width:128px;height:128px;"></div>
```

Comment:
Keep styling in CSS rather than inline in HTML. It's nice to keep these separated whenever you can.

Reply:
Got it. However, whenever I was struggling to resize the image in CSS and keep its proportions without turning it into a background image, so the HTML option seemed to be my only way to go. Do you have any recommendations?

Reply:
That's interesting - the style tag should be identical to adding those same styles in CSS. Maybe we could look at the specifics at some point and I can help debug.